src/atlantis/common/rpc.go

```diff
@@ -81,35 +86,13 @@ func (r *RPCClient) newTLSClient() (*rpc.Client, error) {
 	return c, err
 }
 
-func (r *RPCClient) checkVersion() error {
-	if r.VersionOk {
-		return nil
-	}
-	arg := VersionArg{}
-	var reply VersionReply
-	err := r.doRequest("Version", arg, &reply)
-	if err != nil {
-		r.VersionError = err
-		r.VersionOk = false
-		return err
-	}
-	if !CompatibleVersions(reply.RPCVersion, r.RPCVersion) {
-		err := errors.New("Version Mismatch. Server: " + reply.RPCVersion + ", Client: " + r.RPCVersion)
-		r.VersionError = err
-		r.VersionOk = false
-		return err
-	}
-	r.VersionOk = true
-	return nil
-}
-
-func (r *RPCClient) checkVersionWithTimeout(timeout int) error {
+func (r *RPCClient) checkVersion(region int) error {
```

Comment:
Well, it's private to the package, and never used, so I assume not.

I actually deleted it accidentally, but then nothing broke, so I left it deleted.  (Though it should have been a separate commit.)